tutorial/notebooks/introduction.ipynb

```diff
@@ -18,26 +18,25 @@
+    "\n",
+    "\n",
+    "In the second mode applications have to be re-writen in a Domain-Specific Language (DSL) embedded in Fortran.\n",
     "DSLs are a potential solution to the problem of supporting the scientific development of complex codes that must achieve high performance on a range of different high performance computers. The way in which DSLs achieve this is to separate the science code from its parallelisation and optimisation. This separation allows for single source science codes (which helps with scientific productivity) that can run efficiently (performance) on different HPC architectures (portability).\n",
     "\n",
     "PSyclone does not support a single DSL, rather it can be configured to support different DSL front ends (called APIs in PSyclone). For example, the finite element domain deals with concepts such as function spaces and basis functions whereas the finite difference domain deals with concepts such as Arakawa grids and grid-point types. Therefore the DSLs for these different domains will also differ.\n",
     "\n",
-    "Three APIs are currently supported in PSyclone. These are called \"nemo\", \"lfric\" (which was previously named \"lfric\") and \"gocean\".\n",
-    "\n",
-    "* nemo supports 3D, structured finite-difference codes. As its name suggests, it was initially developed to support the NEMO ocean model but it is now also being used in work with the NEMOVAR, CROCO and ROMS codes.\n",
+    "PSyclone currently supports two PSyKAl DSLs: \"lfric\" and \"gocean\".\n",
     "\n",
     "* lfric supports mixed, finite-element codes and is used in the LFRic Project, which is developing the Met Office's next generation weather and climate model.\n",
     "\n",
     "* gocean supports 2D, structured finite-difference codes. It is more of a development API but has been demonstrated in the MOST Tsunami code.\n",
     "\n",
     "\n",
-    "All three APIs take Fortran code as input and can produce optimised parallel code as output. The input Fortran code is parsed using the fparser2 parser (a lightweight parser implemented in Python) and then a language-independent description of the code to be generated is constructed in the PSyclone Internal Representation (PSyIR). It is the PSyIR that is manipulated in order to optimise the code\n",
+    "All two APIs take Fortran code as input and can produce optimised parallel code as output. The input Fortran code is parsed using the fparser2 parser (a lightweight parser implemented in Python) and then a language-independent description of the code to be generated is constructed in the PSyclone Internal Representation (PSyIR). It is the PSyIR that is manipulated in order to optimise the code\n",
```

Comment:
An oddity of English but "Both" instead of "All two"

Reply:
Oh yes, fixed